tsconfig.json

```diff
@@ -22,6 +22,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "views/homepage/OrganizationsSections"],
```

Comment:
no

Reply:
I didn't even add this to be honest, I have no clue how it got there or why but I'll remove it now